docs/docs/gatsby-cli.md

```diff
@@ -28,15 +28,11 @@ Now we have the `develop` script available to be used which will use our package
 
 ## How to use
 
-By using [npx](https://www.npmjs.com/package/npx) you can utilize the Gatsby command line tool without installing it globally. This lets you scaffold a project with just one command!
-
-For example you can run `npx gatsby new your-site-name` to generate a new Gatsby site.
-
-Run `npx gatsby --help` for full help.
+Run `gatsby --help` for full help.
```

Comment:
This should reference globally installing it first and foremost!

Reply:
@DSchau You got it. Will update 🙌